config/api.yml

```diff
@@ -4744,6 +4744,8 @@
         :identifier: vm_guest_shutdown
       - :name: reboot_guest
         :identifier: vm_guest_restart
+      - :name: set_description
+        :identifier: vm_rename
```

Comment:
Thought the `vm_rename` feature would work for changing the description, but LMK if you want a new product feature for this